README.md

```diff
@@ -71,19 +79,29 @@ $ sudo systemctl enable openvswitch.service
 $ sudo systemctl start openvswitch.service
 </pre>
 b. Add ns3.py into mininet <br/>
-<pre>$ cp $on/mininet-patch/mininet/ns3.py $mn/mininet</pre>
+<pre>
+$ cp $on/mininet-patch/mininet/ns3.py $mn/mininet
+</pre>
 c. Replace mininet/node.py with the one in mininet-patch <br/>
-<pre>$ cp $on/mininet-patch/mininet/node.py $mn/mininet</pre>
+<pre>
+$ cp $on/mininet-patch/mininet/node.py $mn/mininet
+</pre>
 d. Add WiFi roaming simulation script to example <br/>
-<pre>$ cp $on/mininet-patch/examples/wifiroaming.py $mn/examples</pre>
+<pre>
+$ cp $on/mininet-patch/examples/wifiroaming.py $mn/examples
+</pre>
 e. Rebuild Mininet <br/>
-<pre>$ sudo util/install.sh -n</pre>
+<pre>
+$ sudo util/install.sh -n
+</pre>
 
 3. Install, patch and Build ns-3 <br/>
 a. Copy patches to the ns-3.21 folder <br/>
 <pre>
 $ cd $ns/ns-3.21
-`$ cp $on/ns3-patch/*.patch .`
+$ cp $on/ns3-patch/animation-interface.patch .
```

Comment:
I suggest we keep the original one in case there are more patches needed in the future.